code/obj/item/misc_weapons.dm

```diff
@@ -2130,6 +2130,64 @@ obj/item/whetstone
 #undef HALB_MED_STAMCOST
 #undef HALB_LIGHT_STAMCOST
 
+/obj/item/crashaxe
+	name = "crash axe"
+	desc = "A lightweight utility-survival axe designed to cut through pod hulls and other structures. Works just as well on bothersome crewmembers."
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "crashaxe"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	hitsound ='sound/impact_sounds/coconut_break.ogg' //shamelessly re-using this sound idea from my halberd
+
+	health = 5 //same item health as most tools
+	tool_flags = TOOL_CUTTING | TOOL_CHOPPING
+
+	w_class = W_CLASS_SMALL
+	hit_type = DAMAGE_CUT
+	leaves_slash_wound = TRUE //yeah prolly
+	force = 14
+	stamina_damage = 25
+	throwforce = 24
+	throw_speed = 4
+	throw_range = 7
```

Comment:
https://github.com/goonstation/goonstation/pull/21354#pullrequestreview-2441260993

Fraction of the size of a katana and only deals one less damage. Now has it even has swipe.
Throwforce is one less than an axe, has greater range, greater speed and has a special knockdown/disorient.

Reply:
hm yea good point

Reply:
If those are the bar, then 200 points seems exceptionally cheap for an item of that caliber.  That is cheaper than the sledgehammer.